### PR TITLE
Translate Section 6 to French

### PR DESCRIPTION
**Pull request by jeffmaher**
# Description

[Trello card:](https://trello.com/c/a46aCfHK/309-people-can-access-prototype-in-en-and-fr) Translates section 6 of the medical report into French

# Test Instructions

1. Go to the application (from the linked "View Deployment")
1. Change the URL to `.../fr/frsection6` from `.../en/section6`)
1. See French translation (original EN and FR forms are linked in the [Trello card](https://trello.com/c/a46aCfHK/309-people-can-access-prototype-in-en-and-fr))

# Help Requested

If there are any things you spot that are issues we should come back to, but are out of scope of just the direct translation, please add a new Trello card or [use this one to quickly add a note to the checklist](https://trello.com/c/WqJVeqsh/356-misc-content-tasks). I've already added a few from section 6 to that card, around the conditional questioning.

### CODE REVIEW

**Comment by jeffmaher** [comment]
@keithiopia : Could you review? Ping back here if updates are needed. If things look OK, assign to Dave for code review.

**Comment by keithiopia** [comment]
> @keithiopia : Could you review? Ping back here if updates are needed. If things look OK, assign to Dave for code review.

I might not have time today 😬 If it can wait till tomorrow, then I'm all over it.

File: locales/en.json [hunk 1]
```diff
@@ -114,27 +114,26 @@
...
-  "No (skip to Section 7)": "No (skip to Section 7)",
-  "Unknown (skip to Section 7)": "Unknown (skip to Section 7)",
-  "If yes, please indicate <strong>when</strong> you expect your patient to return to work": "If yes, please indicate <strong>when</strong> you expect your patient to return to work",
-  "In 6 to 12 months": "In 6 to 12 months",
-  "In 12 to 24 months": "In 12 to 24 months",
-  "In more than 24 months": "In more than 24 months",
+  "section6.title": "Section 6 - Patient's employment situation",
+  "section6.instructions": "This section gathers information to assess current and future restrictions on the patient's ability to work.",
+  "section6.stop_work_question": "Did you recommend to your patient that they stop working?",
+  "section6.stop_work_not_discussed": "section6.stop_work_not_discussed",
```
Comment (CalvinRodo) [inline locales/en.json]: Missing english text

**Review by CalvinRodo — CHANGES_REQUESTED** [review]

File: locales/en.json (hunk 1 above)
Comment (jeffmaher) [reply locales/en.json]: Got it! Added in.

**Comment by jeffmaher** [comment]
@keithiopia : No prob. @sidewalkballet : Do you have time to take over?

**Review by courtneycl — DISMISSED** [review]
LGTM. I added a checklist entry in the misc tasks trello card about the validation needing to be translated.

**Comment by jeffmaher** [comment]
Back over to you @CalvinRodo

**Review by CalvinRodo — DISMISSED** [review]

**Comment by jeffmaher** [comment]
I'm working on merge conflicts

**Comment by jeffmaher** [comment]
All merged in

**Review by CalvinRodo — APPROVED** [review]
🉑

**Pull request merged**